examples/em-jsontemplates/ABTestingTemplate.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
+    "experiment_name": "Compare performance objective of A/B software releases",
+    "mode": "Apply/MetricCollectionOnly",
+    "environment": "QA/PROD",
+    "resource": {
+      "namespace": "Default",
+      "deployment_name": "tfb-qrh-sample"
+    },
+    "datasource_info": {
+      "prometheus1": {
+        "url": "http://10.101.144.137:9090",
+        "provider": "prometheus"
+      }
+    },
+    "settings": {
+      "trial_settings": {
+        "measurement_cycles": "3",
+        "warmup_duration": "1min",
+        "warmup_cycles": "3",
+        "measurement_duration": "1min",
+        "iterations": "3"
+      },
+      "deployment_settings": {
+        "deployment_policy": {
+          "type": "rollingUpdate"
+        }
+      }
+    },
+    "pod_metrics": {
+      "request_sum": {
+        "datasource": "prometheus1",
+        "query": "rate(http_server_requests_seconds_sum{method=\"GET\",outcome=\"SUCCESS\",status=\"200\",uri=\"/db\",}[1m])",
+        "name": "request_sum"
+      },
+      "request_count": {
+        "datasource": "prometheus1",
+        "query": "rate(http_server_requests_seconds_count{method=\"GET\",outcome=\"SUCCESS\",status=\"200\",uri=\"/db\",}[1m])",
+        "name": "request_count"
+      }
+    },
+    "container_metrics": {
+      "container_name": {
+        "MaxInlineLevel": {
```

Comment:
`jvm_memory_used_bytes` is no longer associated with `MaxInineLevel` but with `GC`